src/sentry/integrations/vsts/vsts_subscription_script.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+import logging
+from uuid import uuid4
+
+from sentry.constants import ObjectStatus
+from sentry.integrations.exceptions import ApiError, ApiUnauthorized
+from sentry.models import OrganizationIntegration
+
+logger = logging.getLogger('sentry.integrations.vsts_script')
+
+
+def create_webhook(integration, organization_id):
+    installation = integration.get_installation(organization_id)
+    client = installation.get_client()
+    shared_secret = uuid4.hex + uuid4.hex
+
+    resp = client.create_subscription(
```

Comment:
This is the wrong function signature, and I don't know what it needs to be. When I ran this, I got:

```
TypeError: create_subscription() takes exactly 4 arguments (3 given)
```